.github/workflows/preview.yml

```diff
@@ -12,7 +12,7 @@ concurrency: preview-${{ github.ref }}
 
 jobs:
   deploy-preview:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
This should be removed, especially as you changed it to "latest" in your other pr. Is this preventing anything in this PR from deploying/building?